### PR TITLE
Feature/search result app initial setup

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFSOEG-130

#### Description

This PR adds the initial setup of the search result app.
Right now the search results are just shown as titles.
The pager is added but is unstyled.
We need this setup in order to be able to work simultaneously in the team with the sub components inside the Search Result App.

#### Screenshot of the result

image

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Review by Adamik10 — APPROVED** [review]
I am very impressed how you just produced all this code out of thin air. I have a comment or two, but it's approved from my side. Great job!

File: src/apps/search-result/search-result.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          searchItemsShown={searchItemsShown}
+          hitcount={hitcount}
+          setPageHandler={setPageHandler}
+        />
+      )}
+    </>
+  );
+};
+
+export default SearchResult;
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/apps/search-result/search-result.tsx:81]: (I mean the whole file)

File: src/apps/search-result/search-result.dev.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { withQuery } from "@storybook/addon-queryparams";
+import SearchResultEntry, {
+  SearchResultEntryProps
+} from "./search-result.entry";
+
+export default {
+  title: "Apps / Search Result",
+  component: SearchResultEntry,
+  decorators: [withQuery],
+  parameters: {
```
[the remark this answers is not in the document]
Comment (spaceo) [reply src/apps/search-result/search-result.dev.tsx:12]: We do not need to change it. It is used by the storybook queryparams-addon

File: src/apps/search-result/search-result.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    setSearchItemsShown(currentPage * numberOfResultItems);
+  };
+
+  const { isSuccess, data } = useSearchWithPaginationQuery({
+    q: { all: q },
+    offset: page === 1 ? 0 : searchItemsShown,
+    limit: numberOfResultItems
+  });
+
+  const firstWork = data?.search?.works?.[0]?.id;
```
Comment (sinejespersen) [inline src/apps/search-result/search-result.tsx:40]: maybe a comment to explain these consts?

**Review by sinejespersen — APPROVED** [review]

File: src/apps/search-result/search-result.tsx (hunk 3 above)
Comment (spaceo) [reply src/apps/search-result/search-result.tsx:41]: Yes! Good point.

**Pull request merged**